internal/engine/upper_test.go

```diff
@@ -967,6 +967,42 @@ k8s_yaml('snack.yaml')`
 	f.assertAllBuildsConsumed()
 }
 
+func TestConfigChange_ManifestIncludingInitialBuildsIfTriggerModeChangedToManualAfterInitial(t *testing.T) {
+	f := newTestFixture(t)
+	defer f.TearDown()
+
+	foo := f.newManifest("foo").WithTriggerMode(model.TriggerModeManualIncludingInitial)
+	bar := f.newManifest("bar")
+
+	f.Start([]model.Manifest{foo, bar}, true)
+
+	// foo should be skipped, and just bar built
+	call := f.nextCallComplete("initial build")
+	require.Equal(t, bar.ImageTargetAt(0), call.firstImgTarg())
+
+	// since foo is "Manual", it should not be built on startup
+	// make sure the "queue" is empty
```

Comment:
ughhhh, yeah

fwiw, when reading recent trigger PRs I've been really confused because in my head "queued" means "waiting to build", and we have vars like "isQueued" that just mean "in the trigger queue", so they're false even though the manifests waiting to build